xcode/Subconscious/Shared/Parsers/SubtextUtilities.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SubtextUtilities.swift
+//  Domain-specific extensions to Subtext.
+//  Subconscious
+//
+//  Created by Gordon Brander on 12/5/23.
+//
+
+import Foundation
+
+extension Subtext {
+    /// Read all valid slashlinks from DOM as an array of `Slashlink`.
+    var parsedSlashlinks: [Subconscious.Slashlink] {
+        slashlinks.compactMap({ subtextSlashlink in
+            Subconscious.Slashlink(String(describing: subtextSlashlink))
+        })
+    }
+}
```

Comment:
Extension to read proper `Slashlink` instead of the markup `Subtext.Slashlink` (which is just a marked range in a string).

Written as extension to keep app domain logic out of the Subtext parser.